db/migrate/20240729103525_update_proxy_byos_column_type.rb

```diff
@@ -0,0 +1,13 @@
+class UpdateProxyByosColumnType < ActiveRecord::Migration[6.1]
+  def up
+    add_column :systems, :proxy_byos_mode, :integer, default: 0
+    safety_assured do
```

Comment:
will this work with millions of systems? Or is batched more safe here?

Reply:
good point, I added [838d16a](https://github.com/SUSE/rmt/pull/1193/commits/838d16a2cc1de4797adbcf5f1ef9bedbd16947a0) for the batches approach

Reply:
good point, [0b62c2b](https://github.com/SUSE/rmt/pull/1193/commits/0b62c2b1820767bb2e15543a0b6885699ea3f1ef) addresses the batch approach